src/main/java/util/LineUtils.java

```diff
@@ -0,0 +1,16 @@
+package util;
+
+public final class LineUtils {
+
+    private LineUtils() {
+        throw new IllegalStateException();
+    }
+
+    public static String[] convertToArray(String query) {
+        if (query.equals("")) {
+            throw new IllegalArgumentException("You press the enter");
```

Comment:
Please use a better exception message. For example, "You pass the wrong query".

Reply:
See https://github.com/Ksef/simple-search-engine/pull/1/files#r784260639

Reply:
used this exception message, thx)